README.md

```diff
@@ -1,4 +1,4 @@
-![Awesome Books Project](assets/awesome_books_full_website.png)
+![Awesome Books Project](assets/img/awesome_books_full_website.png)
```

Comment:
- [ ] On your readme file, please delete the "live demo" section along with its contents, since in this milestone is not required, and there is a link that does not provide any live demo.

That way, your project will be well documented 💪

Reply:
Thank you 
I learn a lot from that